webui/cypress/integration/features/user.feature

```diff
@@ -0,0 +1,11 @@
+Feature: User Management
+
+  Scenario: Manage User
```

Comment:
Can you also add a scenario to test that a user who is not an admin should not be able to edit permissions?

Reply:
added